src/WorkspaceDependencyViewExtension/Properties/Resources.resx

```diff
@@ -117,231 +117,246 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="DetailsMessageIncorrectVersion" xml:space="preserve">
+    <value>Some nodes in this graph were originally created with {0} v{1} which you do not have installed. Instead you have a different version of {0} installed, would you keep the installed version?</value>
+  </data>
   <data name="DetailsMessageLoaded" xml:space="preserve">
-    <value>{0} {1} is installed locally.</value>
+    <value>{0} {1} is installed and loaded.</value>
   </data>
   <data name="DetailsMessageMissing" xml:space="preserve">
     <value>This graph depends on {0} {1}. Would you like to download and install this package?</value>
   </data>
   <data name="DetailsMessageWarning" xml:space="preserve">
-    <value>Some nodes in this graph were originally created with {0} {1} which you do not have installed locally. You may have a different version of {0} installed, or you may have a different package which resolves these nodes.</value>
+    <value>Some nodes in this graph were originally created with {0} v{1} which you do not have installed. You may have a different version of {0} installed, or you may have a different package which resolves these nodes.</value>
```

Comment:
`Some nodes in this graph were originally created with {0} v{1}, which you do not have installed. `